web/src/utilities/utiltyFunctions.ts

```diff
@@ -17,3 +17,35 @@ export function postGisToPoint(location: string): Point | undefined {
   }
   return { x: xCoord, y: yCoord };
 }
+
+export function radiusForZoomLevel(currentZoom: number, minZoom: number, maxZoom: number): number {
+  const inRangeRadius = 10;
+  let diff = 0;
+  if (currentZoom >= minZoom && currentZoom <= maxZoom) {
```

Comment:
I would restructure the sequence of `if` statements to be "if-else if-else" instead--this will improve readability and it will more strongly express the disjoint nature of the test. I am thinking something like `if (currentZoom < minZoom)`, then `else if (currentZoom > maxZoom)`, then `else` (to handle the "in range" portion of the test).

Reply:
`eslint` actually gets mad if I turn these into `if/else` statements, since I'm returning a value within the `if` block.

Reply:
In that case, you're still better off doing something like setting up a `radius` variable, setting it in the cascade of if-else if-else tests, and returning a single value at the end.

Generally speaking, you should prioritize readability and idiomatic usage (which this change would be), and satisfy the linter under those constraints.